docs/CreateEdit.md

```diff
@@ -147,51 +147,14 @@ Using a custom `EditActions` component also allow to remove the `<DeleteButton>`
 
 ## Prefilling a `<Create>` Record
 
-By default, the `<Create>` view starts with an empty `record`. You can pass a custom `record` object to start with preset values:
+You may need to prepopulate a record based on a related record. For instance, in a `PostShow` component, you may want to display a button to create a comment related to the current post. Clicking on that button would lead to a `CommentCreate` page where the `post_id` is preset to the id of the Post.
 
-```jsx
-const commentDefaultValue = { nb_views: 0 };
-export const CommentCreate = (props) => (
-    <Create {...props} record={commentDefaultValue}>
-        <SimpleForm>
-            <TextInput source="author" />
-            <RichTextInput source="body" />
-            <NumberInput source="nb_views" />
-        </SimpleForm>
-    </Create>
-);
-```
-
-While using the `record` to set default values works here, it doesn't work with `<Edit>`. So it's recommended to use [the `defaultValue` prop in the Form component](#default-values) instead.
-
-However, there is a valid use case for presetting the `record` prop: to prepopulate a record based on a related record. For instance, in a `PostShow` component, you may want to display a button to create a comment related to the current post, that would lead to a `CommentCreate` page where the `post_id` is preset.
+By default, the `<Create>` view starts with an empty `record`. However, if the `location` object (injected by [react-router](https://reacttraining.com/react-router/web/api/location)) contains a `record` in its `state`, the `<Create>` view uses that `record` instead of the empty object.
```

Comment:
Ok so you're making my not yet published first tutorial on advanced react-admin obsolete already! :angry: